includes/class-wc-stripe-webhook-handler.php

```
@@ -1599,48 +1599,86 @@ public function process_checkout_session_completed( $notification ): void {
-		WC_Stripe_Logger::info(
-			'Agentic checkout.session.completed received.',
-			[
-				'session_id'        => $checkout_session->id ?? 'unknown',
-				'payment_intent_id' => $payment_intent_id ?? 'unknown',
-			]
-		);
+		if ( ! $session->is_agentic() ) {
+			return;
+		}
+
+		$payment_intent_id = $session->get_payment_intent_id();
```

PHPMD (2.15.0)

[error] The variable $payment_intent_id is not named in camelCase. (undefined)

(CamelCaseVariableName)